stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -195,6 +198,14 @@ data class PaymentSessionConfig internal constructor(
             this.shouldShowGooglePay = shouldShowGooglePay
         }
 
+        /**
+         * @param canDeletePaymentMethods if `false`, user could not remove payment method
+         * by swiping right on the selected payment method in [PaymentMethodsActivity]
```

Comment:
maybe reword to `@param canDeletePaymentMethods controls whether the user can delete a payment method by swiping on it in PaymentMethodsActivity. Defaults to true.`